simde/arm/neon/cgt.h

```diff
@@ -45,12 +45,13 @@ simde_vcgt_f32(simde_float32x2_t a, simde_float32x2_t b) {
       b_ = simde_float32x2_to_private(b);
     simde_uint32x2_private r_;
 
-    #if defined(SIMDE_VECTOR_SUBSCRIPT_OPS)
+    #if defined(SIMDE_VECTOR_SUBSCRIPT_OPS) && 0
+      /* These need a cast to avoid warning-errors but I am not sure how to cast it :-( */
```

Comment:
`HEDLEY_REINTERPRET_CAST`.  You can even cheat and do `HEDLEY_REINTERPRET_CAST(__typeof__(r_.values), a_.values > b_.values)` since every compiler that supports GCC-style vector extensions also supports `__typeof__` :)